plugins/core/jetbrains-community/src/migration/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileWatcher.kt

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package migration.software.aws.toolkits.jetbrains.core.credentials.profiles
+
+import com.intellij.openapi.components.service
+
+interface ProfileWatcher {
```

Comment:
Can we add a TODO here to ensure that this gets implemented only in regular IDEs and not in GW?

Reply:
why would we do that? we're getting rid of gateway